lib/gems/pending/appliance_console/cli.rb

```diff
@@ -118,11 +126,13 @@ def parse(args)
         opt :ipapassword,   "IPA Server password",  :type => :string
         opt :ipadomain,     "IPA Server domain (optional)", :type => :string
         opt :iparealm,      "IPA Server realm (optional)", :type => :string
-        opt :ca,                   "CA name used for certmonger",       :type => :string,  :default => "ipa"
-        opt :postgres_client_cert, "install certs for postgres client", :type => :boolean
-        opt :postgres_server_cert, "install certs for postgres server", :type => :boolean
-        opt :http_cert,            "install certs for http server",     :type => :boolean
-        opt :extauth_opts,         "External Authentication Options",   :type => :string
+        opt :datetime,             "DateTime in YYYY-MM-DDTHH:MM:SS format", :type => :string
+        opt :autosync,             "Automatic time synchronization",         :type => :boolean
```

Comment:
ok, time sync is False by fedault in the appliance, I'll make the change for on or off